aodn_cloud_optimised/config/dataset/anmn_hourly_timeseries.json

```diff
@@ -1,22 +1,16 @@
 {
   "dataset_name": "anmn_hourly_timeseries",
-  "logger_name": "anmn_hourly_timeseries",
```

Comment:
a bit annoying that the order changed, it's harder to review and see the changes